node/config/settings/base.py

```diff
@@ -59,13 +66,15 @@
     },
 ]
 
+MONGO_HOST = os.getenv(f'{ENVVAR_SETTINGS_PREFIX}MONGO_HOST', '127.0.0.1')  # type: ignore # noqa: F821
```

Comment:
we are not doing it. every variable is already overridable with `TNB_<VARNAME>=<json string>`

Reply:
Yes, I find way how to override host IP address with "mongo" domain name.
`TNB_DATABASES: '{"default": {"CLIENT": {"host": "mongo"}}}'`